functorch/test/test_vmap.py

```diff
@@ -3277,7 +3277,9 @@ def test():
              {torch.float32: tol(atol=1e-04, rtol=1e-02)}, device_type='cuda'),
     ))
     @toleranceOverride({torch.float32: tol(atol=1e-04, rtol=1e-04)})
-    @skipOps('TestVmapOperatorsOpInfo', 'test_vmap_exhaustive', vmap_fail)
+    @skipOps('TestVmapOperatorsOpInfo', 'test_vmap_exhaustive', vmap_fail.union({
+        xfail('cat'),
+    }))
```

Comment:
This removes vmap testing for torch.cat entirely, leaving us prone to regressions slipping in. In the future, please ping someone who works on functorch if you're adding something to the expected failure lists or skipping the tests

Reply:
Hi, Sure, @zou3519! Thank you for your points. Will keep that in my mind and ping you or someone working on functorch. I'm sorry about the disruptions caused.